src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  padding: 50px 40px;
+
+  --column-count: 1;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
```

Comment:
The media query for 488px seems to be an unusual breakpoint. Typically, breakpoints are set at standard device widths such as 480px, 768px, 1024px, etc. Consider adjusting this to a more common breakpoint unless there is a specific reason for using 488px.